### PR TITLE
Added a section for cost categories in dashboards

### PR DESCRIPTION
**Pull request by charanya-j**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by abhijeet-harness — APPROVED** [review]

**Review by trunapushpa — APPROVED** [review]

File: docs/cloud-cost-management/2-use-cloud-cost-management/3-ccm-cost-categories/use-ccm-cost-categories.md [hunk 1]
```diff
@@ -132,7 +137,31 @@ When you change the definition of the Cost Category, it automatically changes wh
 
 For example, if a new product is added to the Manufacturing department, you can simply update the Manufacturing bucket in the Departments Cost Category, and that change is automatically reflected in all the Perspectives that use that Cost Category.
 
-### See Also
+## Use Cost Categories in Dashboards
+
+You can include cost categories while creating visualizations in your custom dashboard. To learn how to create a custom dashboard, go to [Create Dashboards](/docs/platform/18_Dashboards/create-dashboards.md).
```
Comment (Rohit-Kaliki) [inline docs/cloud-cost-management/2-use-cloud-cost-management/3-ccm-cost-categories/use-ccm-cost-categories.md:142]: Cost Categories automatically flow into dashboards, you can not choose to include/ exclude.

File: docs/cloud-cost-management/2-use-cloud-cost-management/3-ccm-cost-categories/use-ccm-cost-categories.md [hunk 2]
```diff
@@ -132,7 +137,31 @@ When you change the definition of the Cost Category, it automatically changes wh
...
+
+**Cost Categories** is available in the Unified explore on the Dashboards page. 
+
+  <img src={dashboards_cc} alt="A screenshot to show the unified explore." height="400" width="500" />
+
+
+* When you create a new cost category or make changes to an existing one, it may take up to 24 hours for the changes to be reflected in the dashboard data.
+* If you are using AWS or Azure, cost categories are applied to data fetched from the current month onwards through CUR and Billing Exports. It is not applied to historical data. On the other hand, if you are using GCP, the cost category is applied to data that is equal to or more recent than the last three days' data.
```
Comment (Rohit-Kaliki) [inline docs/cloud-cost-management/2-use-cloud-cost-management/3-ccm-cost-categories/use-ccm-cost-categories.md:150]: Cost Category changes/ updates

Comment (Rohit-Kaliki) [inline docs/cloud-cost-management/2-use-cloud-cost-management/3-ccm-cost-categories/use-ccm-cost-categories.md:150]: Cost Category changes/ updates are

File: docs/cloud-cost-management/2-use-cloud-cost-management/3-ccm-cost-categories/use-ccm-cost-categories.md [hunk 3]
```diff
@@ -132,7 +137,31 @@ When you change the definition of the Cost Category, it automatically changes wh
...
+* When you delete a cost category, the deleted category remains visible in the dashboard until the end of the month, as it is applied to data for the ongoing month. For instance, if you delete a cost category on January 24th, the category is visible in the dashboard until the end of January 31st.
+
+:::note
+In AWS, you cannot apply cost categories if you have selected any of the following fields in the explore:
```
Comment (Rohit-Kaliki) [inline docs/cloud-cost-management/2-use-cloud-cost-management/3-ccm-cost-categories/use-ccm-cost-categories.md:154]: You can not use cost category as a dimension in custom dashboards if you have selected .....

File: docs/cloud-cost-management/2-use-cloud-cost-management/3-ccm-cost-categories/use-ccm-cost-categories.md [hunk 4]
```diff
@@ -132,7 +137,31 @@ When you change the definition of the Cost Category, it automatically changes wh
...
+
+- Resource ID 
+- Line Item Type 
+- Market Type 
+- Amortised Cost 
+- Net Amortised Cost 
+
+:::
+
+## See Also
```
Comment (Rohit-Kaliki) [inline docs/cloud-cost-management/2-use-cloud-cost-management/3-ccm-cost-categories/use-ccm-cost-categories.md:164]: Cost Categories behaviour in Perspectives vs Custom Dashboards
 We need to add a section for this? 

When a cost category changes - perspectives always reflect the current state of the cost category definition for all historical data. But the behaviour in dashboards is as you mentioned ^

**Review by Rohit-Kaliki — CHANGES_REQUESTED** [review]

File: docs/cloud-cost-management/2-use-cloud-cost-management/3-ccm-cost-categories/use-ccm-cost-categories.md [hunk 5]
```diff
@@ -23,15 +28,15 @@ If there are items that don't belong to any of these cost categories, the cost a
...
+## Prerequisites and Permissions
 
 To use Cost Categories, your Harness User account must belong to a User Group with the following Role permissions:
 
 * **Cloud Cost Management**: **Cost Categories**: **Create/Edit**
 
 For more details, go to [CCM Roles and Permissions](../../4-cloud-cost-technical-reference/ccm-ref/ccm-roles-and-permissions.md).
 
-### Visual Summary
+## Visual Summary
```
Comment (SmritiSatya) [inline docs/cloud-cost-management/2-use-cloud-cost-management/3-ccm-cost-categories/use-ccm-cost-categories.md:40]: Visual summary

File: docs/cloud-cost-management/2-use-cloud-cost-management/3-ccm-cost-categories/use-ccm-cost-categories.md [hunk 6]
```diff
@@ -23,15 +28,15 @@ If there are items that don't belong to any of these cost categories, the cost a
 
 You can include Cost Categories while creating visualizations in your custom dashboard. To learn how to create a custom dashboard, go to [Create Dashboards](/docs/platform/18_Dashboards/create-dashboards.md).
 
-### Prerequisites and Permissions
+## Prerequisites and Permissions
```
Comment (SmritiSatya) [inline docs/cloud-cost-management/2-use-cloud-cost-management/3-ccm-cost-categories/use-ccm-cost-categories.md:32]: Prerequisites and permissions

**Review by SmritiSatya — DISMISSED** [review]

**Pull request merged**